CONTRIBUTING.md

```diff
@@ -1,62 +1,99 @@
 # Contributing Guidelines
 
-Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
-documentation, we greatly value feedback and contributions from our community.
+Thank you for your interest in contributing to our project. We greatly value
+feedback and contributions from our community!
 
-Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
-information to effectively respond to your bug report or contribution.
+Reviewing this document will maximize your success in working with the
+codebase and sending pull requests.
 
 ## Getting Started
 
+### Find things to do
+
+If you're looking for ideas about where to contribute, consider
+[_good first issue_](https://github.com/aws/aws-toolkit-vscode/labels/good%20first%20issue)
+issues.
+
 ### Setup
 
-Before you start, install the following dependencies.
+To develop this project, install these dependencies:
```

Comment:
Maybe include SAM cli and Docker for dependencies? Could put it as [Optional] for working with lambda features, noting that some tests will fail without these tools.